### PR TITLE
drainer/*: get latest timestamp from pd when initial-commit-ts is -1

### PR DESCRIPTION
**Pull request by july2993**
### What problem does this PR solve? 
easy deploy for starting at the latest timestamp

### What is changed and how it works?
drainer/*: get the latest timestamp from PD when initial-commit-ts is -1

### Check List 

Tests 

 - Integration test

Code changes

Side effects

Related changes
 - Need to cherry-pick to the release branch
 - Need to update the documentation
 - Need to update the `tidb-ansible` repository
 - Need to be included in the release note

### CODE REVIEW

File: drainer/config.go [hunk 1]
```diff
@@ -123,7 +123,7 @@ func NewConfig() *Config {
 	fs.StringVar(&cfg.MetricsAddr, "metrics-addr", "", "prometheus pushgateway address, leaves it empty will disable prometheus push")
 	fs.IntVar(&cfg.MetricsInterval, "metrics-interval", 15, "prometheus client push interval in second, set \"0\" to disable prometheus push")
 	fs.StringVar(&cfg.LogFile, "log-file", "", "log file path")
-	fs.Int64Var(&cfg.InitialCommitTS, "initial-commit-ts", 0, "if drainer donesn't have checkpoint, use initial commitTS to initial checkpoint")
+	fs.Int64Var(&cfg.InitialCommitTS, "initial-commit-ts", 0, "if drainer donesn't have checkpoint, use initial commitTS to initial checkpoint, will get a latest timestamp from pd if setting to be -1")
```
Comment (ericsyh) [inline drainer/config.go]: ### ### 
```suggestion
	fs.Int64Var(&cfg.InitialCommitTS, "initial-commit-ts", -1, "if drainer donesn't have checkpoint, use initial commitTS to initial checkpoint, will get a latest timestamp from pd if setting to be -1")
```

Comment (ericsyh) [reply drainer/config.go]: Set `initial-commit-ts` = -1 by default?

**Comment by ericsyh** [comment]
LGTM

**Comment by GregoryIan** [comment]
please fix CI @july2993

**Comment by july2993** [comment]
@suzaku @GregoryIan PTAL

**Comment by july2993** [comment]
@suzaku pls approve

**Pull request merged**